### PR TITLE
Created Slide Component

### PR DESCRIPTION
**Pull request by RickyWang1234**
created event slide first page where arrows are clickable but have no interactions thus far

### CODE REVIEW

File: src/components/team/Teammate.jsx [hunk 1]
```diff
@@ -1,5 +1,37 @@
+import React from 'react';
```
Comment (pondohoo) [inline src/components/team/Teammate.jsx:1]: make sure you're not changing unrelated files within different issues (this is the slides issue but you're changing Teammate.jsx)

File: src/components/events/slidesAlignment.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
+.container {
```
Comment (pondohoo) [inline src/components/events/slidesAlignment.css:1]: only use tailwind inline styles, don't use default css

**Review by pondohoo — COMMENTED** [review]
make sure to include an image of your work running on a page and link the related issue under "Development"

**Comment by RickyWang1234** [comment]
issue 12 screnshot

**Review by pondohoo — DISMISSED** [review]

File: src/components/events/Slides.jsx [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+
 const Slides = () => {
-  return <div>Slides</div>;
+  return (
+    <div className="Slides">
+      <div className="flex justify-center items-center">
+        <div className="my-55 mr-40">
+          <AiOutlineArrowLeft
+            size={50}
+            className="text-asme-blue-600 cursor-pointer text-xl"
```
Comment (pondohoo) [inline src/components/events/Slides.jsx:11]: bro it's ok to copy but don't get caught in 4k. change the blue 😭

File: src/components/events/Slides.jsx [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+          />
+        </div>
+
+        <div className="center section">
+          <div className="font-playfair text-lg pt-4">
```
Comment (pondohoo) [inline src/components/events/Slides.jsx:16]: don't specify any font

File: src/components/events/Slides.jsx [hunk 5]
```diff
@@ -1,5 +1,38 @@
...
+            alt="registration line Image"
+            width={600} // Desired width of the image
+            height={200} // Desired height of the image
+          />
+        </div>
+
+        <div className="my-55 ml-40">
+          <AiOutlineArrowRight
+            size={50}
+            className="text-asme-blue-600 cursor-pointer text-xl"
```
Comment (pondohoo) [inline src/components/events/Slides.jsx:30]: blue

File: src/components/events/Slides.jsx [hunk 6]
```diff
@@ -1,5 +1,38 @@
...
+          />
+        </div>
+
+        <div className="center section">
+          <div className="font-playfair text-lg pt-4">
+            <p>Event Name</p>
+          </div>
+          <Image
+            src="registration-line.png" // Path to your image inside the 'public' directory
```
Comment (pondohoo) [inline src/components/events/Slides.jsx:20]: import as an object instead of specifying width and height

i.e. something like
import RegistrationLine from registration-line.png
...
<Image
src={RegistrationLine}

File: src/components/events/Slides.jsx [hunk 7]
```diff
@@ -1,5 +1,31 @@
...
+  return (
+    <div className="Slides">
+      <div className="flex justify-center items-center">
+        <div className="my-55 mr-40">
+          <AiOutlineArrowLeft size={50} className="cursor-pointer text-xl" />
+        </div>
+
+        <div className="center section">
+          <div className="text-lg pt-4">
+            <p>Event Name</p>
```
Comment (pondohoo) [inline src/components/events/Slides.jsx:15]: p tag m-0 p-0

**Review by pondohoo — APPROVED** [review]
